packages/components/src/Chip/Chip.css

```diff
@@ -5,11 +5,11 @@
   --chip-radius: 20px;
   --chip-base-bg: var(--color-surface--background);
   --chip-base--hover: var(--color-surface--background--hover);
-  display: flex;
+  display: inline-flex;
   position: relative;
   height: var(--chip-height);
   min-width: 0;
-  padding: var(--space-small) var(--base-unit);
```

Comment:
What's this change coming from?

Reply:
disclaimer: I can't speak to most of this since I'm really the reviewer and not the author

my best guess is that it's not needed because we're doing `align-items: center` on the parent with a flex direction of row so it gets centered vertically

<img width="352" alt="image" src="https://github.com/user-attachments/assets/efc88679-c858-4811-9fa5-860d732e114c">